src/content/banner.astro

```diff
@@ -19,6 +19,7 @@ import BearLink from "../components/BearLink.astro";
 </div>
 <style>
   .banner {
+    background-color: #0a1126;
```

Comment:
I need to re-export the banner from figma and make sure it works well on widescreens. meanwhile, you can use `var(--color-primary)` here to match the color from the dark blue sections

Reply:
merging this to not create conflicts when I change the cover image

Reply:
Oh cool, yeah feel free to change this. I scanned the banner svg file and found that reference, it would be cool if figma exported the svg with css variables references in it 👁️ no idea if that's even possible honestly.